### ISSUE TITLE
Duplicates command palettes with VS Code

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode-edge-devtools#396]
I showed the Edge Devtools and js-debug intregration today. Something that came up was the fact that there's two command palettes; if you ctrl+shift+p in the devtools editor, you get a different palette than you do in VS Code.

It would be nice to integrate the commands from the devtools into VS Code command's palette, or at least use quickpicks, for a more consistent experience.

**Comment by codepo8** [comment]
Makes sense, adding this as a scenario - the only thing I wonder is how that works for screenreaders and keyboard users.

**Comment by connor4312** [comment]
The command palette in VS Code is accessible. @isidorn is the local accessibility wizard who can answer any more specific questions you might have 🙂

**Comment by isidorn** [comment]
Yup, let me know if you have any questions :)
